.github/workflows/openbsdvm.yaml

```diff
@@ -0,0 +1,30 @@
+name: OpenBSD VM
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - openbsd-vm-workflow
+    paths-ignore:
+      - '**.md'
+      - '**.yml'
+
+
```

Comment:
wouldn't have commented if this was the only thing, but let's lose one of the two newlines. I'll let you pick which one. :-)